### PR TITLE
fix(FX-4315): hide artsy guarantee in some cases

### PR DESCRIPTION
**Pull request by gkartalis**
The type of this PR is: **Fix**

This PR solves [FX-4315]

### Description

Hides Artsy guarantee section when an artwork is:

- sold
- in auction
- isOfferableFromInquiry: false

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FX-4315]: https://artsyproduct.atlassian.net/browse/FX-4315?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2.tsx [hunk 1]
```diff
@@ -79,6 +80,12 @@ export const ArtworkSidebar2: React.FC<ArtworkSidebarProps> = ({
 
   const lotLabel = artwork.isBiddable ? artwork.saleArtwork?.lotLabel : null
 
+  const shouldDisplayArtsyGuarantee =
+    !checkIfArtworkIsOnLoanOrPermanentCollection(artwork.saleMessage) &&
+    !isSold &&
+    !isInAuction &&
+    isOfferableFromInquiry
```
Comment (dblandin) [inline src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2.tsx:87]: **question**: Will this be `true` even if `isInquireable=false`?

**Review by dblandin — APPROVED** [review]
LGTM but left a question about the inquireable logic.

File: src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2.tsx (hunk 1 above)
Comment (gkartalis) [reply src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2.tsx:87]: as far as I understood isInquireable is unrelated, but will add some reviewers from NX just to make sure

**Review by araujobarret — APPROVED** [review]

**Comment by gkartalis** [comment]
thanks @araujobarret

**Pull request merged**